biojava-structure/src/main/java/org/biojava/nbio/structure/io/mmcif/SimpleMMcifConsumer.java

```diff
@@ -1816,6 +1879,71 @@ public void newStructConn(StructConn structConn) {
 		this.structConn.add(structConn);
 	}
 
+	void createSSBonds() {
```

Comment:
It is good practice to explicitly set the visibility of the method, it really helps others understanding the code and helps readability in general. We've tried to do it throughout BioJava as much as we could. I'd set it to private unless otherwise needed.